### ISSUE TITLE
No notification sounds and 3 volume sliders for whatsapp-for-linux

### ISSUE THREAD

**Issue by bkmo** [eneshecan/whatsapp-for-linux#212]
Arch Linux
KDE Plasma 5.25
Pipewire wireplumber audio

There are 3 volume sliders created with the app active, but no notification sounds are heard
![whatsapplinux](https://user-images.githubusercontent.com/777373/179552520-7c0d3d28-481e-4182-8cd5-4b6824dde365.png)

**Comment by SayanChakroborty** [comment]
Hi @eneshecan. I am reopening this issue because the duplicate you linked i.e. #173 does not mention the issue of having 3 volume sliders. It might not look like a big issue until you use [Application Volume Mixer](https://extensions.gnome.org/extension/3499/application-volume-mixer) extension on Gnome desktop to show volume sliders for individual applications. While using Whatsapp for Linux, Gnome System Tray gets convoluted and really long with multiple volume sliders for this app but serving no purpose at all. My question is why have 3 volume sliders when none of them serve any purpose and if possible how do I disable them?

**Comment by eneshecan** [comment]
I don't think we can do much about that since webkit creates different audio channels for media, notifications etc. It's the same behavior with all browsers. However, it's worth having a look.
